src/browser/modules/Stream/CypherFrame/__snapshots__/VisualizationView.test.tsx.snap

```diff
@@ -5,178 +5,151 @@ exports[`Visualization renders 1`] = `<div />`;
 exports[`Visualization renders with result and escapes any HTML 1`] = `
```

Comment:
Not sure if these snapshots give us anything really. Might be a candidate for removal in the future.

Reply:
It's been discussed in the past and we've looking to remove them in cooldown